### PR TITLE
Remove support for games as a media type

### LINKED ISSUE #593: Remove Games
**Is your feature request related to a problem? Please describe.**
The games service is half-assed, requires a separate plugin and manual install steps and is basically a glorified gallery given that no clients really support it. It should be removed as it's not even in the spirit of Jellyfin.

**Describe the solution you'd like**
Remove https://github.com/jellyfin/jellyfin/blob/dev/MediaBrowser.Api/GamesService.cs and related classes.

**Describe alternatives you've considered**
There are none.

**Additional context**
Not even the upstream project really believes in this feature: https://emby.media/community/index.php?/topic/62513-36037-giving-games-some-love/?p=619211

### PR DESCRIPTION
**Pull request by cvium**
**Changes**
Wiped support for Games from the core.

Frontend PR: https://github.com/jellyfin/jellyfin-web/pull/80
**Issues**
Fixes #593

### CODE REVIEW

File: Emby.Server.Implementations/Emby.Server.Implementations.csproj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
```
Comment (JustAMan) [inline Emby.Server.Implementations/Emby.Server.Implementations.csproj]: what is this change?

File: MediaBrowser.Api/MediaBrowser.Api.csproj [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
```
Comment (JustAMan) [inline MediaBrowser.Api/MediaBrowser.Api.csproj]: and what changed here?

File: MediaBrowser.Controller/MediaBrowser.Controller.csproj [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
```
Comment (JustAMan) [inline MediaBrowser.Controller/MediaBrowser.Controller.csproj]: and this.... what changed?

File: MediaBrowser.LocalMetadata/MediaBrowser.LocalMetadata.csproj [hunk 4]
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
```
Comment (JustAMan) [inline MediaBrowser.LocalMetadata/MediaBrowser.LocalMetadata.csproj]: and this...

File: MediaBrowser.Model/MediaBrowser.Model.csproj [hunk 5]
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
```
Comment (JustAMan) [inline MediaBrowser.Model/MediaBrowser.Model.csproj]: and this...

File: MediaBrowser.Providers/MediaBrowser.Providers.csproj [hunk 6]
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
```
Comment (JustAMan) [inline MediaBrowser.Providers/MediaBrowser.Providers.csproj]: and this

**Comment by cvium** [comment]
@JustAMan I'm assuming my IDE (Rider) just followed the editorconfig...

File: MediaBrowser.Api/UserLibrary/BaseItemsRequest.cs [hunk 7]
```diff
@@ -42,12 +42,6 @@ protected BaseItemsRequest()
         [ApiMember(Name = "MinIndexNumber", Description = "Optional filter by minimum index number.", IsRequired = false, DataType = "int", ParameterType = "query", Verb = "GET")]
         public int? MinIndexNumber { get; set; }
 
-        [ApiMember(Name = "MinPlayers", Description = "Optional filter by minimum number of game players.", IsRequired = false, DataType = "int", ParameterType = "query", Verb = "GET")]
```
Comment (JustAMan) [inline MediaBrowser.Api/UserLibrary/BaseItemsRequest.cs]: What bugs me here is this is backwards-incompatible change which, if we strictly follow semver, should make us change version to `11.0.0`... maybe better add some stubs returning some dummy values? cc @joshuaboniface

Comment (cvium) [reply MediaBrowser.Api/UserLibrary/BaseItemsRequest.cs]: I get what you're saying, but I think it was previously discussed that we will have breaking changes here and there, but once we reach a stable API we should release 11.0.0 (or higher).

I prefer to break shit early and not mock anything unnecessarily. It's just tech debt that no one remembers later.

Comment (JustAMan) [reply MediaBrowser.Api/UserLibrary/BaseItemsRequest.cs]: Okay, let's leave it like that for now.
I doubt any client except Web UI ever cared about games anyway :)

**Pull request merged**